src/main/java/mate/academy/dao/MovieSessionDao.java

```diff
@@ -6,9 +6,13 @@
 import mate.academy.model.MovieSession;
 
 public interface MovieSessionDao {
-    List<MovieSession> findAvailableSessions(Long movieId, LocalDate date);
+    MovieSession add(MovieSession movieSession);
 
     Optional<MovieSession> get(Long id);
 
-    MovieSession add(MovieSession session);
+    List<MovieSession> findAvailableSessions(Long movieId, LocalDate date);
+
+    Boolean update(MovieSession movieSession);
+
+    Boolean delete(Long id);
```

Comment:
Consider using the primitive `boolean` instead of `Boolean` for the `delete` method's return type to avoid unnecessary boxing and potential nullability issues.